### PR TITLE
Fix log out error

### PR DESCRIPTION
**Pull request by jjennifergu**
## Overview

Bug fixes for log out function

## Error Screenshot
Crashes when logging out

Screen Shot 2023-03-25 at 10 18 47 PM

### CODE REVIEW

File: Eatery Blue/Eatery Blue/UI/Profile/ProfileViewController.swift [hunk 1]
```diff
@@ -76,12 +76,16 @@ class ProfileViewController: UIViewController {
         } else {
             viewControllers[0] = viewController
         }
-        theNavigationController.setViewControllers(viewControllers, animated: animated)
```
Comment (NgaiJustin) [inline Eatery Blue/Eatery Blue/UI/Profile/ProfileViewController.swift:79]: This variable naming is a bit weird. Slight nit but I think consistent naming is important. Either it is a specific navigation control or you can just call it `navigationController`. 
You can learn more about the style guidelines (under naming).

https://www.swift.org/documentation/api-design-guidelines/

File: Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift [hunk 2]
```diff
@@ -12,7 +12,7 @@ import CoreLocation
 
 class HomeModelController: HomeViewController {
     
-    private var isTesting = true
```
Comment (NgaiJustin) [inline Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift:15]: If this testing variable is no longer needed, you can consider removing it and any associated logic from the vc.

**Review by NgaiJustin — APPROVED** [review]
Yup looks good just left a few comments. Also if the future trajectory requires it, you can consider looking into feature flags rather than embedding these boolean variables into the code (which need to be removed later). There is a pod called "FeatureFlags" which you can check out.

You can specify the flags in a separate JSON loaded in when you build the project (you also gain a slight performance increase). You can look into more how these are down in this article

https://medium.com/@rwbutler/feature-flags-a-b-testing-mvt-on-ios-718339ac7aa1

**Review by antoinztte — APPROVED** [review]
👍🏼

File: Eatery Blue/Eatery Blue/UI/Profile/ProfileViewController.swift (hunk 1 above)
Comment (ReadeP) [reply Eatery Blue/Eatery Blue/UI/Profile/ProfileViewController.swift:79]: +1

**Review by ReadeP — APPROVED** [review]

**Pull request merged**